frontend/src/pages/events/index.js

```diff
@@ -6,33 +6,86 @@ import Instagram from "../../components/instaBar";
 import SundayCelebBarEvents from "../../components/page-events/sundayCelebBarEvents";
 import { StaticImage } from "gatsby-plugin-image";
 import EventCard from "../../components/page-events/eventCard";
-import Banner from "../../components/shared/banner";
+// import Banner from "../../components/shared/banner";
```

Comment:
Why did you comment out Banner?

Reply:
it was not being used so i commented it. do you want me to uncomment it?

Reply:
based upon the changes you made it is being used: 
You remove the Banner Component for the EventPage.  Look at the Code you removed and replaced with.

Reply:
hi dom ive fixed this issue and i think it should be able to be merged into main